src/core/server/integration_tests/saved_objects/migrations/zdt_v2_compat/jest.integration.config.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+module.exports = {
+  // TODO replace the line below with
+  // preset: '@kbn/test/jest_integration_node
+  // to do so, we must fix all integration tests first
+  // see https://github.com/elastic/kibana/pull/130255/
+  preset: '@kbn/test/jest_integration',
+  rootDir: '../../../../../../..',
+  roots: ['<rootDir>/src/core/server/integration_tests/saved_objects/migrations/zdt_v2_compat'],
+  // must override to match all test given there is no `integration_tests` subfolder
```

Comment:
I added a dedicated suite for the v2 compat tests. The reasoning was that we may eventually want to get rid of the support later, so having dedicated test suites would make the removal easier.

Reply:
I've been thinking the same thing: Isolate our newest changes in new CI groups.